cleanup-oracle.sh

```diff
@@ -192,12 +192,11 @@ fi
 
 export ANSIBLE_NOCOWS=1
 
-ANSIBLE_PLAYBOOK=`which ansible-playbook 2> /dev/null`
+ANSIBLE_PLAYBOOK="ansible-playbook"
+ANSIBLE_PLAYBOOK_CHECK=`ansible-playbook --version 2> /dev/null`
 if [ $? -ne 0 ]; then
```

Comment:
While we're editing this anyways, I think there's an opportunity to make the check a bit more readable:

https://github.com/koalaman/shellcheck/wiki/SC2181

Reply:
Done